recipes/jupyter-project/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "jupyter-project" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|replace("-", "_") }}-{{ version }}.tar.gz
+  sha256: 379bf82a815c5be5896b8b8103013cae8ba5bd83d738d2d8d3901aa36e49dcf2
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - jupyter-packaging
+    - jupyterlab ~=1.2
+    - nodejs
+    - python ~=3.5
```

Comment:
What is this constraint? We do not ship python 3.5

Reply:
This is a shortcut for `python >=3.5,<4`. But I can transform it like that if it fits better the standard. 

The reason for the python constrain is that as this is a `noarch` package, I just want to be safe rather than sorry ;)

Reply:
This means python >=3.5,<3.6 IIUIC, right?

Reply:
Ohhh wait sorry. I always forget if they add an extra version or not.